vignettes/docker_and_codespaces.Rmd

```diff
@@ -24,6 +24,8 @@ This article will go over the following:
 * What are codespaces, why we are using it for Admiral, and how to use them.
 * How codespaces are built (this section concerns more admiralci developers to be able to maintain codespaces - it's not needed to use codespaces, but interesting for users that want to learn more about admiral CI/CD processes).
 
+Go [here](https://www.youtube.com/watch?v=VUGeXqz_TXo&list=PLn-mqs_tEh86kA-UzWHeVZWIQNm6qvr_z&index=14) to watch codespaces demo !
+
 ## What are codespaces and how to use them
 
 ### What are codespaces, and why are we using it for Admiral ?
```

Comment:
I would always write "Admiral" or "admiral" as `{admiral}` for consistency (across all pages)

Reply:
typo below: "contributors" instead of "contributor"